docs/installation/download-datasets.md

```diff
@@ -35,13 +39,17 @@ included in the docker image.
 To get the ObjectNav dataset and precomputed distance caches for `RoboTHOR` run the following command:
 
 ```bash
+cd datasets
 sh datasets/download_navigation_datasets.sh robothor-objectnav
+cd ../
 ```
 This will download the dataset into `datasets/robothor-objectnav`.
 
 ### iTHOR
 To get the ObjectNav dataset and precomputed distance caches for `iTHOR` run the following command:
 ```bash
+cd datasets
```

Comment:
I could actually download the pointnav datasets to the right location without having to change directories